android/build.gradle

```diff
@@ -10,13 +10,21 @@ buildscript {
 
 apply plugin: 'com.android.library'
 
+def _ext = rootProject.ext
+
+def _reactNativeVersion = _ext.has('reactNative') ? _ext.reactNative : '+'
```

Comment:
Wouldn't it make sense to have the setting be named `reactNativeVersion` for consistency?